src/components/navigation/navbar.css

```diff
@@ -0,0 +1,66 @@
+/* Position and sizing of burger button */
+.bm-burger-button {
+  position: fixed;
+  width: 36px;
+  height: 30px;
+  left: 36px;
+  top: 36px;
+}
+
+/* Color/shape of burger icon bars */
+.bm-burger-bars {
+  background: #373a47;
+}
+
+/* Color/shape of burger icon bars on hover*/
+/* .bm-burger-bars-hover {
+  background: #a90000;
+} */
+
+/* Position and sizing of clickable cross button */
+.bm-cross-button {
+  height: 24px;
+  width: 24px;
+}
```

Comment:
The cross button is a bit small. Ideally it would be bigger.

Reply:
I made it a bit bigger and darker, but let me know if you want it even larger.

Reply:
This looks good as it is now